src/cobald/daemon/core/config.py

```diff
@@ -138,10 +138,10 @@ def translate_hierarchy(self, structure, *, where="", **construct_kwargs):
             prev_item, items = None, []
             for index, item in reversed(list(enumerate(pipeline))):
                 if prev_item is not None:
-                    try:
+                    if hasattr(item, '__rshift__'):
```

Comment:
Please change this to double-quotes to match the ``black`` code style:
```suggestion
                    if hasattr(item, "__rshift__"):
```